### ISSUE TITLE
Debug version of contracts not working.

### ISSUE THREAD

**Issue by lenzj** [nrc/libhoare#25]
The "debug_precond", "debug_postcond", and "debug_invariant" contracts are still present when compiled in release mode.

Seems to be related to line 288 in lib.rs.
```
|item| item.node == ast::MetaWord(token::get_name(token::intern("ndebug")))) {
```
The ndebug flag has been replaced with "debug_assertions" from what I could find online.  I replaced the ndebug string with debug_assertions in that line, but it still doesn't seem to work and I'm not sure how to fix the other portions of that section of code.

**Comment by nrc** [comment]
Should be fixed by 804956a3c3f4e1021b2340ff773012f1e5a6487f (need to test it...)

**Comment by nrc** [comment]
Actually it probably won't

**Comment by nrc** [comment]
Fixed by 901973ba7034559a72b44dbd2b64f78ddde20c6f
